go/rpc/encrypted_client.go

```diff
@@ -133,9 +135,19 @@ func (c *EncRPCClient) Subscribe(ctx context.Context, namespace string, ch inter
 		return nil, fmt.Errorf("expected a channel of type `chan *types.Log`, got %T", ch)
 	}
 
-	// TODO - #453 - Map incoming filters.FilterCriteria to a common.LogSubscription.
+	accountSignature, err := crypto.Sign(c.Account().Hash().Bytes(), c.viewingKey.PrivateKey.ExportECDSA())
```

Comment:
this section could do with a comment, or even be moved to an explicit method.

Reply:
Done